pymc3/distributions/discrete.py

```diff
@@ -27,6 +27,20 @@ class Binomial(Discrete):
 
     .. math:: f(x \mid n, p) = \binom{n}{x} p^x (1-p)^{n-x}
 
+    .. plot::
+        import matplotlib.pyplot as plt
+        import numpy as np
+        import scipy.stats as st
+        plt.style.use('seaborn-darkgrid')
```

Comment:
I think it is better to remove `plt.style.use('seaborn-darkgrid')` and keep the style the same as the ones in `continuous.py` (eg, using a `lambda` function and `plot_pdf`)

Reply:
I agree we should use the same style for continuous and discrete variables, but I prefer this one.